plugins/20_events/install

```diff
@@ -13,30 +13,30 @@ flag_rsyslog_needs_restart=n
 mkdir -m 775 -p "$DOKKU_LOGS_DIR"
 case "$DOKKU_DISTRO" in
   arch|debian|centos)
-    chgrp dokku "$DOKKU_LOGS_DIR"
+    chgrp "$DOKKU_USER" "$DOKKU_LOGS_DIR"
     ;;
   *)
-    chown syslog:dokku "$DOKKU_LOGS_DIR"
+    chown "syslog:$DOKKU_USER" "$DOKKU_LOGS_DIR"
     ;;
 esac
 
 if [[ ! -f  "$DOKKU_EVENTS_LOGFILE" ]]; then
   touch "$DOKKU_EVENTS_LOGFILE"
   case "$DOKKU_DISTRO" in
     arch|debian|centos)
-      chgrp dokku "$DOKKU_EVENTS_LOGFILE"
+      chgrp "$DOKKU_USER" "$DOKKU_EVENTS_LOGFILE"
       ;;
     *)
       # chown syslog:root might not work on SUSE
-      chown syslog:dokku "$DOKKU_EVENTS_LOGFILE"
+      chown "syslog:$DOKKU_USER" "$DOKKU_EVENTS_LOGFILE"
       ;;
   esac
   chmod 664 "$DOKKU_EVENTS_LOGFILE"
 fi
 
 if [[ -d "$DOKKU_RSYSLOG_FILTER_DIR" && ! -f  "$DOKKU_RSYSLOG_FILTER" ]]; then
   cat >"$DOKKU_RSYSLOG_FILTER" <<EOF
-:syslogtag, contains, "dokku" $DOKKU_EVENTS_LOGFILE
+:syslogtag, contains, "$DOKKU_USER" $DOKKU_EVENTS_LOGFILE
```

Comment:
If I'm recalling rsyslog correctly, I think that has the potential to route other logs to this file if the configured user owns other processes.